proto-public/pbmesh/v1alpha1/proxy_configuration.proto

```diff
@@ -93,6 +93,7 @@ message BootstrapConfig {
   string stats_config_json = 11;
   string stats_flush_interval = 12;
   string tracing_config_json = 13;
+  string telemetry_collector_bind_socket_dir = 14;
```

Comment:
Assuming this was necessary in another context since it wasn't explicitly used in this PR.